### ISSUE TITLE
Disappear when dragging near to center bottom

### ISSUE THREAD

**Issue by dingbo8128** [shadabunique/flutter_draggable_fab#1]

**Comment by shadabunique** [comment]
I tried and not able to reproduce the issue.
Can you please add some more description(like device info) or attach a video supporting the same issue?

**Comment by nullbytesoftware** [comment]
it just disappears everytime after you let go

**Comment by endromeda** [comment]
Yes I am Also facing the same issue. When i drag buttons right top and release buttons then it disappears.

**Comment by shadabunique** [comment]
Still facing the issue? Please provide a video/screenshot with the device details. 
That will help figure out the issue as I can't reproduce on my end although trying different devices.

**Comment by cherishloveyou** [comment]
> 

after  init drag move it  and it disappear

**Comment by cherishloveyou** [comment]
> Still facing the issue? Please provide a video/screenshot with the device details. That will help figure out the issue as I can't reproduce on my end although trying different devices.

appear in release mode below iPhone8, I guess  Size screenSize = MediaQuery.of(context).size;  is not the exact  screen size

**Comment by yawboafo** [comment]
Yes i'm facing the same issue.
